src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.cxx

```diff
@@ -113,6 +113,224 @@ t8_geometry_squared_disk::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreei
   }
 }
 
+/**
+ * Map the faces of an oktaeder to a spherical surface.
+ * \param [in]  cmesh      The cmesh in which the point lies.
+ * \param [in]  gtreeid    The global tree (of the cmesh) in which the reference point is.
+ * \param [in]  ref_coords  Array of \a dimension many entries, specifying a point in [0,1]^dimension.
+ * \param [out] out_coords  The mapped coordinates in physical space of \a ref_coords.
+ */
+void
+t8_geometry_triangulated_spherical_surface::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid,
+                                                              const double *ref_coords, const size_t num_coords,
+                                                              double *out_coords) const
+{
+  double n[3]; /* Normal vector of the current triangle. */
```

Comment:
Please use more explaining names, like normal, radial, position for example.

Reply:
Done.